var/spack/repos/builtin/packages/abinit/package.py

```diff
@@ -186,7 +186,7 @@ def configure_args(self):
                 oapp(f"F90={spec['mpi'].mpifc}")
 
             # MPI version:
-            # let the configure script auto-detect MPI support from mpi_prefix
+            # let the confgiure script auto-detect MPI support from mpi_prefix
```

Comment:
Can you undo the modification to this spelling?

Reply:
OOPS!!! (WTH happened?!?) :)